### ISSUE TITLE
inclusion of metadata, especially accessibility metadata

### ISSUE THREAD

**Issue by RachelComerford** [w3c/publ-cg#53]
from BISG survey

**Comment by dauwhe** [comment]
Is https://idpf.github.io/epub-vocabs/package/a11y/ still a thing?

**Comment by avneeshsingh** [comment]
Indeed. This metadata is critical for reporting conformance and certification. 
Both the conformance tools from DAISY Consortium, Ace and SMART are using it.
Certification agencies in DAISY network use it, and ingestion/distribution systems like Vital Source are updating their systems to consume it.

In fact, a parallel discussion happened in PWG for ensuring that it can also flow in WP/PWP/EPUB 4.

**Comment by mattgarrish** [comment]
Some of the relevant links here include:
- http://www.idpf.org/epub/a11y/#sec-disc-package
- https://www.w3.org/wiki/WebSchemas/Accessibility
- http://www.a11ymetadata.org
- http://kb.daisy.org/publishing/docs/metadata/schema-org.html

**Comment by RachelComerford** [comment]
@dauwhe I'm guessing we need to migrate that IDPF content though?
